### PR TITLE
Updates for UFL (helpdesk #6979)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 1]
```diff
@@ -128,10 +128,6 @@ Resources:
         Primary:
           ID: 2d9012f731159dd45a082164c6aac6577167e725
           Name: Bockjoo Kim
-      Submitter Contact:
-        Primary:
-          ID: dcc5225ec8889c92053920bbd934bedadc799908
-          Name: Craig Prescott
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]: Why didn't we replace this with the mailing list?

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 2]
```diff
@@ -196,10 +192,6 @@ Resources:
         Primary:
           ID: 2d9012f731159dd45a082164c6aac6577167e725
           Name: Bockjoo Kim
-      Miscellaneous Contact:
-        Primary:
-          ID: dcc5225ec8889c92053920bbd934bedadc799908
-          Name: Craig Prescott
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]: Same question here

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml (hunk 1 above)
Comment (matyasselmeci) [reply topology/University of Florida/UF HPC/UFlorida-HPC.yaml]: The Submitter is the person who originally submitted the resource, and we can't change that without a time machine :)

He doesn't work there though so there's no point in keeping the entry.

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml (hunk 2 above)
Comment (matyasselmeci) [reply topology/University of Florida/UF HPC/UFlorida-HPC.yaml]: I'll fix that.

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml (hunk 1 above)
Comment (brianhlin) [reply topology/University of Florida/UF HPC/UFlorida-HPC.yaml]: Eh, we have plenty of old submitter info throughout the repos. Let's just keep it around until we decide on the fate of the submitter contacts as a whole.

File: topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml [hunk 3]
```diff
@@ -7,30 +7,26 @@ Resources:
...
-          Name: Yu Fu
+          ID: 8fe996dae2ee3abd17af9f8c0f57fa6d166be5ff
+          Name: University of Florida CMS Tier 2 Admins
         Secondary:
           ID: 2d9012f731159dd45a082164c6aac6577167e725
           Name: Bockjoo Kim
-      Submitter Contact:
-        Primary:
-          ID: dcc5225ec8889c92053920bbd934bedadc799908
-          Name: Craig Prescott
```
Comment (brianhlin) [inline topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml]: Let's also restore this submitter contact removal

**Pull request merged**